### PR TITLE
[mixpanel-browser] - Added optional Options parameter for track_pageview

### PR DESCRIPTION
**Pull request by jarrett-anderson-concertiv**
Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## mixpanel-browser ([<kbd>unpkg</kbd>](https://unpkg.com/browse/mixpanel-browser@latest/))
was missing the following properties:
1. config
1. request_batchers
1. persistence
1. cookie
1. unpersisted_superprops

<details>
<summary>as well as these 8 other properties...</summary>
<p>are_batchers_initialized, init_batchers, start_batch_senders, stop_batch_senders, track_pageview, name_tag, toString, report_error</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a37bd62714ce57de39f98b1fc1655281d60fc9d5
</p>

**Review by eps1lon — CHANGES_REQUESTED** [review]
Please add a link to the documentation for this parameter.

**Comment by jarrett-anderson-concertiv** [comment]
> Please add a link to the documentation for this parameter.

Mixpanel.js Docs
https://github.com/mixpanel/mixpanel-js/blob/master/doc/readme.io/javascript-full-api-reference.md#mixpanel

Mixpanel.js track_pageview function (line 1057)
https://github.com/mixpanel/mixpanel-js/blob/master/src/mixpanel-core.js#L1030C1-L1030C1

File: types/mixpanel-browser/index.d.ts [hunk 1]
```diff
@@ -169,7 +169,7 @@ export interface Mixpanel {
     ): void;
     track_forms(query: Query, event_name: string, properties?: Dict | (() => void)): void;
     track_links(query: Query, event_name: string, properties?: Dict | (() => void)): void;
-    track_pageview(properties?: Dict): void;
+    track_pageview(properties?: Dict, options?: Dict): void;
```
Comment (eps1lon) [inline types/mixpanel-browser/index.d.ts:172]: Shouldn't this be 
```suggestion
    track_pageview(properties?: Dict, options?: { event_name?: string | undefined }): void;
```
according to docs: https://github.com/mixpanel/mixpanel-js/blob/master/doc/readme.io/javascript-full-api-reference.md#mixpaneltrack_pageview?

**Review by eps1lon — APPROVED** [review]

**Review by rbuckton — APPROVED** [review]

**Review by justinhelmer — APPROVED** [review]

**Comment by justinhelmer** [comment]
Ready to merge

**Pull request merged**